runelite-client/src/main/java/net/runelite/client/plugins/dpscounter/DpsOverlay.java

```diff
@@ -114,16 +116,26 @@ public Dimension render(Graphics2D graphics)
 		int maxWidth = ComponentConstants.STANDARD_WIDTH;
 		FontMetrics fontMetrics = graphics.getFontMetrics();
 
-		for (DpsMember dpsMember : dpsMembers.values())
+		for (DpsMember dpsMember : dpsCounterPlugin.getSortedMembers())
```

Comment:
I may reverse my original opinion on moving the sorting out to a plugin, there are too many data races with what you've implemented here, since members is accessed from multiple threads (hence the CHM) and you aren't handling this correctly anywhere currently. Eg here this would NPE if run between member update and sorting. Since there are so few players maybe just sorting here is fine then.